Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MarkdownTextBlock/MarkdownTextBlockPage.xaml.cs

```diff
@@ -50,5 +53,18 @@ private void SetInitalText(string text)
         {
             await Launcher.LaunchUriAsync(new Uri(e.Link));
         }
+
+        private async void MarkdownText_OnImageLoading(object sender, ImageLoadingEventArgs e)
```

Comment:
This should be moved to the documentation instead of the sample app. Otherwise, images load slowly and the user has no idea why.